common-api/build.gradle

```diff
@@ -35,13 +95,28 @@ dependencies {
     exclude module: 'spring-boot-starter-tomcat'
     exclude group: 'org.slf4j', module: 'jul-to-slf4j'
   }
+  compile 'io.swagger:swagger-annotations:1.5.16'
+  compile 'com.google.code.gson:gson:2.8.5'
 
   testCompile 'junit:junit:4.12'
   testCompile 'org.mockito:mockito-core:1.10.19'
   testCompile 'com.google.truth:truth:0.42'
   testCompile 'com.h2database:h2:1.4.194'
   testCompile 'org.liquibase:liquibase-core:3.5.3'
   testCompile 'org.bitbucket.radistao.test:before-after-spring-test-runner:0.1.0'
+
+  // For Swagger generation. These should include dependencies found in the Swagger codegen
```

Comment:
Should we remove these deps from api/build.gradle, then?

Reply:
I don't think so? We are generating code from Swagger specs in both places. If we stopped generating Swagger in common-api, we'd still have these deps in api... so normally I'd say we should spell out the direct dependencies everywhere.

Reply:
On further thought, that is definitely preferred.